index.ts

```diff
@@ -0,0 +1,99 @@
+import R, { values } from 'ramda';
+import { isTemplateSpan } from 'typescript';
+class ObjectWrapper<T extends Object> {
+  private _obj: T;
+  
+  /***
+   * 引数のオブジェクトのコピーを this._objに設定
+   */
+  constructor(_obj: T) {
+    const data = _obj;
+    this._obj = R.clone(data);
+  }
+  /**
+   * this._objのコピーを返却
+   * @return Object
+   */
+  get obj() {
+    const data = this._obj;
+    const cloneObj = R.clone(data);
+    return cloneObj;
```

Comment:
以下のように記述するとすっきり書けます。
```suggestion
    return R.clone(this._obj);
```